command/git.go

```diff
@@ -325,72 +365,21 @@ func (c *gitFetchProject) Execute(ctx context.Context,
 	}
 
 	//Apply patches if necessary
-	if conf.Task.Requester != evergreen.PatchVersionRequester {
-		return nil
-	}
-
-	logger.Execution().Info("Fetching patch.")
-	patch, err := comm.GetTaskPatch(ctx, client.TaskData{ID: conf.Task.Id, Secret: conf.Task.Secret})
-	if err != nil {
-		err = errors.Wrap(err, "Failed to get patch")
-		logger.Execution().Error(err.Error())
-		return err
-	}
-
-	if err = c.getPatchContents(ctx, comm, logger, conf, patch); err != nil {
-		err = errors.Wrap(err, "Failed to get patch contents")
-		logger.Execution().Errorf(err.Error())
-		return err
-	}
-
-	if err = c.applyPatch(ctx, logger, conf, patch); err != nil {
-		err = errors.Wrap(err, "Failed to apply patch")
-		logger.Execution().Infof(err.Error())
-		return err
-	}
-
-	return nil
-}
-
-func parseGitUrl(url string, logger client.LoggerProducer) (string, string) {
```

Comment:
moved this function to the thirdparty package. I wanted to use it elsewhere and it made most sense to locate it there.